### DISCUSSION TITLE
New Code View UI Bug

### DISCUSSION THREAD

**Discussion by meppu** [community/community#53142] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I use new code view which is currently in beta and looks like they added support for repository main page. but the problem is as you can see right side has no padding. it is not a big thing but it looks weird

![image](https://user-images.githubusercontent.com/112294217/232857889-14c51069-40ae-4141-aec0-678b62596ce9.png)

**Comment by Spinoscythe** [comment]
@sicakkanli I don't get this anymore, I want it. Why?

**Comment by meppu** [comment]
![image](https://user-images.githubusercontent.com/112294217/233138322-0eec0a29-dc4b-4b94-b966-7d38f6344c59.png)

looks like they fixed it.
